content/en/tests/test_health.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Test Health
+description: "Measure the impact of flaky tests and how Test Optimization improves CI usage."
+further_reading:
+- link: "tests/flaky_test_management"
+  tag: "Documentation"
+  text: "Learn about managing flaky tests"
+- link: "tests/flaky_test_management/auto_test_retries"
+  tag: "Documentation"
+  text: "Learn about Auto Test Retries"
+- link: "tests/test_impact_analysis"
+  tag: "Documentation"
+  text: "Learn about Test Impact Analysis"
+- link: "tests/flaky_test_management/early_flake_detection"
+  tag: "Documentation"
+  text: "Learn about Early Flake Detection"
+- link: "quality_gates"
+  tag: "Documentation"
+  text: "Learn about Quality Gates"
+---
+
+## Overview
+
+The [Test Health][5] dashboard provides analytics to help teams manage and optimize their testing in CI. This includes sections showing the current impact of test flakiness and how Test Optimization is mitigating these problems.
+
+{{< img src="tests/test-health.png" alt="Test Health dashboard" style="width:100%;" >}}
+
+## Summary metrics
+
+Based on the current time frame and filters applied, the dashboard highlights the following key metrics:
+
+- [**Pipelines Failed**](#pipelines-failed): Sum total of pipelines that failed due to flaky tests
+- [**Time Wasted in CI**](#time-wasted-in-ci): Total time wasted in CI due to flaky tests
```

Comment:
```suggestion
- [**Time Wasted in CI**](#time-wasted-in-ci): Total time spent in CI due to flaky tests
```
Just to avoid echoing the term in the definition